ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2226,6 +2236,12 @@ router_bgp:
       password: "< encrypted_password >"
       send_community: < standard | extended | large | all >
       maximum_routes: < integer >
+      maximum_routes_warning_limit: < integer >
```

Comment:
This model is too static. What we can do was:

maximum_routes_warning_limit: < <integer> | <percent> "percent" >
Let the user specify this either an integer or 10 percent kind of.

Reply:
So the proposal is to change to a string: `< "<integer>" | "<0-100> percent" >`.

Reply:
My goal was to follow the CLI as much as possible,  so either allow the value in percent or integer. In order to not mix types in the value (integer | string) I've introduced the key 

```
maximum_routes_warning_limit_percent: < true | false >
```

would it make sense to convert this into something like 

```
maximum_routes_warning_limit_unit: < percent | integer >
```

This would also allow for different units if they get introduced in the future. Or is it better to just take an arbitrary string like proposed?

Reply:
Changed model to:

```
      maximum_routes: < integer >
      maximum_routes_warning_limit: < "<integer>" | "<0-100> percent" >
      maximum_routes_warning_only: < true | false >
```

If new formats get added, we'll just have to update the documentation accordingly, like Tony mentioned.